engine/apps/email/inbound.py

```diff
@@ -74,11 +85,10 @@ def dispatch(self, request):
         if request.method.lower() == "head":
             return HttpResponse(status=status.HTTP_200_OK)
 
-        integration_token = self.get_integration_token_from_request(request)
-        if integration_token is None:
+        if self.integration_token is None:
```

Comment:
a little bit of refactoring: instead of having a `self.get_integration_token_from_request(request)` method, it's now just a property `self.integration_token`. no functional changes, just looks better IMO